fb/app/models/friendship.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Friendship < ApplicationRecord
+  belongs_to :user
+  belongs_to :friend, class_name: 'User'
+  validates :user_id, presence: true
```

Comment:
[OPTIONAL]
This actually is not necessary.

Reply:
Sure, I will remove this